order.go

```diff
@@ -114,19 +115,30 @@ type DiscountCode struct {
 }
 
 type LineItem struct {
-	ID            int              `json:"id"`
-	ProductID     int              `json:"product_id"`
-	VariantID     int              `json:"variant_id"`
-	Quantity      int              `json:"quantity"`
-	Price         *decimal.Decimal `json:"price"`
-	TotalDiscount *decimal.Decimal `json:"total_discount"`
-	Title         string           `json:"title"`
-	VariantTitle  string           `json:"variant_title"`
-	Name          string           `json:"name"`
-	SKU           string           `json:"sku"`
-	Vendor        string           `json:"vendor"`
-	GiftCard      bool             `json:"gift_card"`
-	Taxable       bool             `json:"taxable"`
+	ID                         int              `json:"id"`
+	ProductID                  int              `json:"product_id"`
+	VariantID                  int              `json:"variant_id"`
+	Quantity                   int              `json:"quantity"`
+	Price                      *decimal.Decimal `json:"price"`
+	TotalDiscount              *decimal.Decimal `json:"total_discount"`
+	Title                      string           `json:"title"`
+	VariantTitle               string           `json:"variant_title"`
+	Name                       string           `json:"name"`
+	SKU                        string           `json:"sku"`
+	Vendor                     string           `json:"vendor"`
+	GiftCard                   bool             `json:"gift_card"`
+	Taxable                    bool             `json:"taxable"`
+	FulfillmentService         string           `json:"fulfillment_service"`
+	RequiresShipping           bool             `json:"requires_shipping"`
+	VariantInventoryManagement string           `json:"variant_inventory_management"`
+	Properties                 []NoteAttribute  `json:"properties"`
+	ProductExists              bool             `json:"product_exists"`
+	FulfillableQuantity        int              `json:"fulfillable_quantity"`
+	Grams                      int              `json:"grams"`
+	FulfillmentStatus          string           `json:"fulfillment_status"`
+	TaxLines                   []TaxLine        `json:"tax_lines"`
+	OriginLocation             Address          `json:"origin_location"`
+	DestinationLocation        Address          `json:"destination_location"`
```

Comment:
I can't find these two location fields in the docs. Where did you get them from?

Reply:
That's the result i got when i tried to fetch my order:
<img width="288" alt="screen shot 2018-01-06 at 20 29 53" src="https://user-images.githubusercontent.com/24430957/34642728-a8526d90-f320-11e7-841f-c97aa5ec6eaa.png">

Reply:
ok interesting. Thanks for confirming. It seems to be a common theme that the documentation is not complete regarding the fields it has.